examples/asr_rnnt_emformer/requirements.txt

```diff
@@ -0,0 +1,3 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torchaudio
```

Comment:
by default, torchserve install it via ts_scripts/install_dependency.py

Reply:
It's fine it's a tutorial